dubbo-dependencies-bom/pom.xml

```diff
@@ -700,6 +701,15 @@
                 <artifactId>grpc-grpclb</artifactId>
                 <version>${grpc.version}</version>
             </dependency>
+            <!-- Groovy, for Embedded Consul. Could be removed once embedded-consul:2.2.1 is released
+             See https://github.com/pszymczyk/embedded-consul/pull/114
+             -->
+            <dependency>
+                <groupId>org.codehaus.groovy</groupId>
+                <artifactId>groovy-xml</artifactId>
+                <version>${groovy.version}</version>
```

Comment:
shall we add it to test scope?

Reply:
good point!
I think it would be good idea to inline `groovy.version`, i.e. to use `<version>2.5.13</version>` 
This way it won't be forgotten in the `<properties>` section when embedded-consul:2.2.1 is released and this explicit dependency removed

Reply:
Done!